distributed/client.py

```diff
@@ -2767,6 +2767,17 @@ def scheduler_info(self, **kwargs):
         self.sync(self._update_scheduler_info)
         return self._scheduler_identity
 
+    def to_scheduler_file(self, scheduler_file):
+        """Dump the info in this file to a scheduler file"""
```

Comment:
Unless we're going to write stand-alone documentation about this somewhere (and this seems small enough to me that we can skip it) we should expand this docstring to include the motivation to use this as well as a small example, maybe like the following:

```python
client = Client()
client.to_scheduler_file('scheduler.json')

client2 = Client(scheduler_file='scheduler.json')  # connect to previous client's scheduler
```